skgstat/Kriging.py

```diff
@@ -88,14 +91,25 @@ def __init__(
             If True, the different parts of the algorithm will record their
             processing time. This is meant to be used for optimization and
             will be removed in a future version. Do not rely on this argument.
+        sparse : bool
+
```

Comment:
Can't we get this information from the Variogram? The internal metric space should know, right?
Or is this referring to something else that is spase? 
Maybe one sentence in the docstring would be helpful